README.md

```diff
@@ -1,8 +1,71 @@
-gatling-maven-plugin-demo-kotlin
-================================
+Gatling plugin for Maven demo project
+=====================================
 
-Simple showcase of a Kotlin maven project using the gatling-maven-plugin.
+A simple showcase of a Maven project using the
+[Gatling plugin for Maven](https://github.com/gatling/gatling-maven-plugin). The full documentation for the plugin can
+be found [on the Gatling website](https://gatling.io/docs/current/extensions/maven_plugin/).
 
-To test it out, execute the following command:
+This project is written in Kotlin, others are available for [Java](https://github.com/gatling/gatling-maven-plugin-demo-java)
+and [Scala](https://github.com/gatling/gatling-maven-plugin-demo-scala).
 
-    $mvn gatling:test
+It includes:
+
+* minimal `pom.xml`
+* latest version of `io.gatling:gatling-maven-plugin` applied
+* sample
+  [Simulation](https://javadoc.io/doc/io.gatling/gatling-core-java/latest/io/gatling/javaapi/core/Simulation.html)
```

Comment:
Same: link to the page generated from https://github.com/gatling/gatling/blob/main/src/docs/content/reference/current/core/concepts/index.md#simulation

Reply:
Changed in all the related pull requests.